Business.Implementation/Validations/CartValidation.cs

```diff
@@ -0,0 +1,16 @@
+﻿using Business.Models;
+
+namespace Business.Implementation.Validations
+{
+    public static class CartValidation
+    {
+        public static void ValidateProduct(CartDto cartDto)
+        {
+
+            if (cartDto.TotalPrice < 0)
+            {
+                throw new ValidationException("Price can't be of negative value");
+            }
+        }
+    }
```

Comment:
Do we really need this cart validation?

Reply:
There was a history of unexpected bugs through development history, like when ghandi goes mad in civilization game series and starts to nuke everyone, so yes, I think it won't be abundant

Reply:
You're fired.